config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,9 @@
+# Sidekiq.configure_client do |config|
```

Comment:
sao lại comment hết rồi???

Reply:
dạ cái này em định để deploy trên heroku 
cơ mà sidekiq dùng redis 4.0.0 
còn thằng redis-to-go đang ở 3.1.2
nên đẩy lên bị lỗi